### PR TITLE
Docs: fix some small grammar / usage issues in the Logging section

### PR DESCRIPTION
**Pull request by sebastianharko**

### CODE REVIEW

**Review by johanandren — APPROVED** [review]
LGTM, thanks @sebastianharko !

**Comment by sebastianharko** [comment]
@johanandren Thanks, I think it's ready to merge.

**Comment by johanandren** [comment]
We require two approvals from the team before merge as a general rule.

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**
